console/src/domain/user/components/PasswordForm.tsx

```diff
@@ -90,15 +98,33 @@ export default function PasswordForm({ currentUser, admin, onSubmit }: IPassword
                         required={passwdValid}
                         validators={[minLength(passwordMinLength, t('Password Too Short'))]}
                     >
-                        <Input type='password' size='compact' disabled={!passwdValid} />
+                        <Input
+                            type='password'
+                            size='compact'
+                            disabled={!passwdValid}
+                            overrides={{
+                                MaskToggleButton: {
+                                    props: { tabindex: -1 },
+                                },
+                            }}
+                        />
                     </FormItem>
                     <FormItem
                         label={t('Confirm New Password')}
                         name='confirmPassword'
                         validators={[shouldBeEqual(() => form.getFieldValue('userPwd'), t('Password Not Equal'))]}
                         required={passwdValid}
                     >
-                        <Input type='password' size='compact' disabled={!passwdValid} />
+                        <Input
+                            type='password'
+                            size='compact'
+                            disabled={!passwdValid}
+                            overrides={{
+                                MaskToggleButton: {
+                                    props: { tabindex: -1 },
+                                },
+                            }}
```

Comment:
Set the `overrides` in the Input component?
https://github.com/star-whale/starwhale/blob/597f16a1c3a8d305502e2b3c57b71228836b391c/console/src/components/Input/Input.tsx#L31